torchreid/utils/torchtools.py

```diff
@@ -307,6 +318,20 @@ def load_pretrained_weights(model, file_path='', pretrained_dict=None):
             'please check the key names manually '
             '(** ignored and continue **)'.format(message)
         )
+
+        if len(discarded_layers) > 0:
```

Comment:
There is a duplication of discarded and unmatched layer names printing: code in this and the next branch is identical. May de move if to a `_*` function?

Reply:
Ok, I will do it

Reply:
Fixed